pkgs/development/libraries/gdk-pixbuf/setup-hook.sh

```diff
@@ -1,17 +0,0 @@
-findGdkPixbufLoaders() {
```

Comment:
One use case that still is ¿not covered? is the use of loaders during `build`/`checkPhase`. Previously `findGdkPixbufLoaders` had us covered but now we need to set the environment variable manually. But maybe it is okay?

Reply:
If we set GDK_PIXBUF_MODULE_FILE for the user, then should we also return to the behaviour of wrapGAppsHook setting it based on the build-time environment variable, so that what works in the build environment also works in the final package?

Reply:
Also, would not being able to control order be an issue?

Reply:
I can imagine some use cases for wanting specific order but they probably be rare so I would not bother with that. People can always juggle with the variable contents if they need it.

Though I am split on having a separate attribute vs. using `buildInputs`. On one hand, separate attribute offers a finer control and the behaviour is more clear by the fact of being explicit. On the other hand all other wrappers just pick stuff up from inputs so not doing that for loaders might be confusing (largely mitigated by docs?) and deciding which dependency to put where might be confusing as well. :woman_shrugging: